python/cudf/cudf/core/tokenize_vocabulary.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+from __future__ import annotations
+
+import cudf
+from cudf._lib.nvtext.tokenize import (
+    TokenizeVocabulary as cpp_tokenize_vocabulary,
+    tokenize_with_vocabulary as cpp_tokenize_with_vocabulary,
+)
+
+
+class TokenizeVocabulary:
+    """
+    A vocabulary object used to tokenize input text.
+
+    Parameters
+    ----------
+    vocabulary : str
+        Strings column of vocabulary terms
+
+    Returns
+    -------
+    TokenizeVocabulary
```

Comment:
No need for `Returns` in a class docstring.

https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring

> Use the same sections as outlined above (all except [Returns](https://numpydoc.readthedocs.io/en/latest/format.html#returns) are applicable).

```suggestion
```